src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,21 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
```

Comment:
Make number 6 a separate variable and use it here and in the loop below

Reply:
Extracted to constant